src/configurator/v1.mli

```diff
@@ -93,7 +93,14 @@ module Pkg_config : sig
      package may contain a version constraint.  For example
      "gtk+-3.0 >= 3.18".  Returns [None] if [package] is not available  *)
 
-  val query_expr : t -> package:string -> expr:string -> package_conf option
+  val query_expr : t
+    -> package:string
+    -> expr:string
+    -> (package_conf, string) Stdune.Either.t
```

Comment:
Umm, exposing `Stdune` this in this interface file may be a problem right?

Reply:
Yes, that's definitely not acceptable. Why don't you just return the result type? This will mean that this will require the result compatibility package in configurator, but I think that's acceptable.

Reply:
Umm, how should such package be exported without exposing `Stdune`?

Reply:
You did exactly what I meant. The result type in stdune matches the one in stdlib so there's no need to export it from stdune.